cypress/integration/AdminSideCMSTest/AdminSideCMSTest.js

```diff
@@ -0,0 +1,99 @@
+// <reference types="Cypress" />
```

Comment:
this is not enabling intelisense  - makes me wonder how long it take to type every line of code without an autosuggest
add another slash in the beginning please, makes you life so much easier
/// <reference types="Cypress" />

Reply:
done